main/services/devon-instances/devon-instances.service.ts

```diff
@@ -104,10 +127,7 @@ export default class DevonInstancesService implements SaveDetails {
       for (let singlepath of paths) {
         if (singlepath) {
           if (process.platform === 'win32') {
-            singlepath = singlepath
-              .replace('/', '')
-              .replace('/', ':/')
-              .replace(/\//g, path.sep);
+            singlepath = this.formatPathToWindows(singlepath);
```

Comment:
Check if you could use nodejs's `path.normalize`: https://nodejs.org/api/path.html#path_path_normalize_path

We are already saving the paths in platform-dependent format using `path.join` (https://nodejs.org/api/path.html#path_path_join_paths) so some replacements might be unnecessary. Please check whether this is the case (paths stored in `data` saved with `path.join`)

If fact, this `if(process.platform === 'win32') ...` might not be needed 
In that case these functions can be removed:
```ts
formatPathToWindows(dirPath: string): string;
formatPathFromWindows(dirPath: string): string;
```

Reply:
The paths saved in `ide-paths` file is always in Linux format irrespective of the platform, hence the need to check if the application is running in windows and then format the paths accordingly (using `formatPathToWindows(dirPath: string): string;`). Else it is unable to understand the path and throws an `ENOENT` error.
`formatPathFromWindows(dirPath: string): string;` is used to update the `ide-paths` file in the same Linux format for consistency.